ncm-nss/src/test/resources/simple.pan

```diff
@@ -0,0 +1,15 @@
+object template simple;
+
+#function pkg_repl = { null; };
```

Comment:
It would be nice to test the schema too by including config.pan but I can't remember why this didn't work now - I wrote the code too long ago. :( What's recommended best practice here?